### PR TITLE
fix:class 'ResultSetUtil.java' parse datetime type error #240

### PR DESCRIPTION
**Pull request by bobobod**

### CODE REVIEW

**Comment by wey-gu** [comment]
I had put this fix PR as hacktoberfest, check https://bit.ly/hack-nebula on how to participate :)

File: src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java [hunk 1]
```diff
@@ -126,27 +127,28 @@ public static <T> T getValue(ValueWrapper valueWrapper, Class<T> resultType) {
   }
 
   private static Object transformDateTime(DateTimeWrapper dateTime) {
+    DateTime localDateTime = dateTime.getLocalDateTime();
```
Comment (Nicole00) [inline src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: Why convert the datetime to localDateTime?
If i am using other zone but not utc8, is there any problems with the results?

Comment (CorvusYe) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: > can we give some description about Why convert the datetime to localDateTime? If i am using other zone but not utc8, is there any problems with the results?

It seems like this.

https://github.com/nebula-contrib/ngbatis/issues/240

Comment (bobobod) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: You'd better test it locally.when the zone is GMT+8, this method of converting time will ultimately result in 8 hours less than expected.

Comment (CorvusYe) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: My nebula-storaged has been unable to start recently and lacks a cross time zone testing environment, so it has been unable to verify. T^T

Comment (bobobod) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: image

Comment (Nicole00) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: @bobobod Does the operating system have different time zone settings?
here is my test:(basic the same as your code)
image

Comment (bobobod) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: I see, The system time zone is right. When I execute the same statement on the nebula console, the result is also correct.

here is my enviroment configuration.
nebula graph: 2.6.2
nebula-java: 3.3.0
image


> @bobobod Does the operating system have different time zone settings? here is my test:(basic the same as your code) image

Comment (bobobod) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: image
This looks like the data returned after calling the service is correct, but after conversion, an error occurred

Comment (Nicole00) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: that's the time zone of the machine where the nebula service is located?

Comment (bobobod) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: it‘s deployed on a virtual machine, the time zone is correct.

Comment (bobobod) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: image

Comment (Nicole00) [reply src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java:130]: I see the point, use LocalDateTime can get the correct time info according to the local zone.

**Review by Nicole00 — APPROVED** [review]
LGTM

**Review by CorvusYe — APPROVED** [review]
LGTM

**Pull request merged**